syntax = "proto3";

package message;

message GetVerifyReq {
    string email = 1;
}

message GetVerifyRsp {
    int32 error = 1;
    string email = 2;
    string code = 3;
}

service VerifyService {
    rpc GetVerifyCode (GetVerifyReq) returns (GetVerifyRsp) {}
}

message GetChatServerReq {
    int32 uid = 1;
}

message GetChatServerRsp {
    int32  error = 1;
    string host = 2;
    string port = 3;
    string token = 4;
}

message CheckTokenReq {
    int32 uid = 1;
    string token = 2;
}

message CheckTokenRsp {
    int32 error = 1;
    int32 uid=2;
    string token = 3;
}

service StatusService {
    rpc GetChatServer (GetChatServerReq) returns (GetChatServerRsp) {}
    rpc CheckToken (CheckTokenReq) returns (CheckTokenRsp) {}
}




